receiver/dockerstatsreceiver/factory.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dockerstatsreceiver
+
+import (
+	"context"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configerror"
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/consumer"
+	"go.uber.org/zap"
+)
+
+const (
+	typeStr = "docker_stats"
+)
+
+var _ component.ReceiverFactoryOld = (*Factory)(nil)
+
+type Factory struct{}
```

Comment:
Looks like this factory is not referenced yet in components.go

Reply:
@bogdandrutu requested I remove it from this PR for the time being